src/main/java/SendMail.java

```diff
@@ -98,70 +92,77 @@ protected PasswordAuthentication getPasswordAuthentication() {
     }
 
     /**
+     * Generates the subject-header.
      *
-     * @param report
+     * @param payload
      * @return
      */
-    private String CreateSubject(Report report){
-        String date = report.getDate().toString();
+    private String CreateSubject(Payload payload){
+        String commithash = payload.getCommitHash();
+        if (commithash == null) {
+            commithash = "Not Found.";
+        }
+
         StringBuilder subjectstring = new StringBuilder();
-        subjectstring.append("Notification for build date: "+ date);
+        subjectstring.append("Notification for buildhash: "+ commithash);
```

Comment:
Maybe replace "build hash" with "build results of commit"

Reply:
Fixed.